app/config/security.yml

```diff
@@ -28,7 +28,7 @@ security:
             anonymous: true
 
         oauth_token:
-            pattern:  ^/api/oauth/v2/token
+            pattern:  ^/api/v1/oauth/v2/token
```

Comment:
Should be reverted. The oauth is a standard so the response won't change until we change the authorization method, but then the path will also change since `/oauth/v2` will no longer be appropriate.